src/main/groovy/liquibase/harness/generateChangelog/GenerateChangelogTestHelper.groovy

```diff
@@ -28,7 +28,7 @@ class GenerateChangelogTestHelper {
         for (DatabaseUnderTest databaseUnderTest : databaseConnectionUtil
                 .initializeDatabasesConnection(TestConfig.instance.getFilteredDatabasesUnderTest())) {
             for (def changeLogEntry : FileUtils.resolveInputFilePaths(databaseUnderTest, baseChangelogPath +
-                    "expectedChangeLog", "sql").entrySet()) {
+                    "expectedChangeLog", "xml").entrySet()) {
```

Comment:
can we use `TestConfig.instance.inputFormat` here? Or maybe we can change to build testInput in other way as we don't really pick up only files with this extension